Source/MethodCallReturn.cs

```diff
@@ -42,7 +42,7 @@
 using System.Globalization;
 using System.Linq.Expressions;
 using System.Reflection;
-
+using System.Runtime.CompilerServices;
```

Comment:
Please move this line a little further up so that `using System.*` are properly sorted. Also, restore the blank line between `using System.*` and `using Moq.*`.

Reply:
About sorting - Runtime is definitely after Reflection, isn't it?
Blank line is my fault - suspect VS dit that :)

Reply:
Uhhh... yes indeed. You're right, of course. :-)